### PR TITLE
Bump Airnode dependencies

### PR DESCRIPTION
**Pull request by aquarat**
This bumps Airnode-related dependencies.

### CODE REVIEW

**Comment by aquarat** [comment]
Clearly bumping the validator was wrong :sweat:

**Comment by vponline** [comment]
Do we need to add the `fulfillmentGasLimit` handling also to `gas-prices.ts` since it's not imported from Airnode?

**Comment by amarthadan** [comment]
If you don't mind I would keep this open until either you or someone else who will be working on [BEC-353](https://api3dao.atlassian.net/browse/BEC-353) finishes it properly. It's mainly missing the support for the new `fulfillmentGasLimit` configuration option. I don't want to add an option to the configuration file and not use it.

**Comment by amarthadan** [comment]
> Do we need to add the `fulfillmentGasLimit` handling also to `gas-prices.ts` since it's not imported from Airnode?

You beat me to it :smile: Yes, we do.

**Comment by acenolaza** [comment]
> If you don't mind I would keep this open until either you or someone else who will be working on [BEC-353](https://api3dao.atlassian.net/browse/BEC-353) finishes it properly. It's mainly missing the support for the new `fulfillmentGasLimit` configuration option. I don't want to add an option to the configuration file and not use it.

I had a little bit of time left today so I've rebased this branch on top of main and added support for `fulfillmentGasLimit` config option ([BEC-353](https://api3dao.atlassian.net/browse/BEC-353))

File: src/gas-prices.ts [hunk 1]
```diff
@@ -18,13 +19,14 @@ export type EIP1559GasTarget = {
   maxFeePerGas: BigNumber;
 };
 
-export type GasTarget = LegacyGasTarget | EIP1559GasTarget;
+export type GasTarget = { gasLimit?: BigNumber } & (LegacyGasTarget | EIP1559GasTarget);
```
Comment (amarthadan) [inline src/gas-prices.ts]: I think the cleaner solution would be something like:
```ts
interface CommonGasTarget {
  gasLimit?: BigNumber;
}

export interface LegacyGasTarget extends CommonGasTarget {
  txType: 'legacy';
  gasPrice: BigNumber;
}

export interface EIP1559GasTarget extends CommonGasTarget {
  txType: 'eip1559';
  maxPriorityFeePerGas: BigNumber;
  maxFeePerGas: BigNumber;
}
```

**Review by amarthadan — APPROVED** [review]

File: src/gas-prices.ts (hunk 1 above)
Comment (Siegrift) [reply src/gas-prices.ts]: This relates to https://api3dao.atlassian.net/browse/AN-618

**Pull request merged**